src/webviews/src/Triggers.tsx

```diff
@@ -26,8 +26,7 @@ const Triggers: FunctionComponent<TriggersProps> = props => {
   return (
     <div className="text-vscode-foreground pb-4">
       <Header title="When to update the data" description="" />
-
-      <CronChooser value={cron} onChange={handleScheduleChange} />
+      {cron && <CronChooser value={cron} onChange={handleScheduleChange} />}
```

Comment:
For whatever reason, `cron` was being passed down as an empty string through the `value` prop which complicated the "is initially valid" check. 

Let's not render `CronChooser` until we have a value.